tests/accumulator/decaying/exponentially.cpp

```diff
@@ -19,9 +19,15 @@ TEST(exponentially_t, Accumulate100OutOf1000Elements) {
     const auto snapshot = accumulator.snapshot();
     EXPECT_EQ(100, snapshot.size());
 
-    for (auto value : snapshot) {
-        ASSERT_TRUE(0 <= value && value <= 1000);
+    for (auto value : snapshot.values()) {
+        ASSERT_TRUE(value <= 1000);
     }
+
+    EXPECT_NEAR(snapshot.mean(), 500, 150);
+    EXPECT_NEAR(snapshot.stddev(), 288, 150);
+
+    EXPECT_NEAR(snapshot.min(), 0, 150);
+    EXPECT_NEAR(snapshot.max(), 1000, 150);
```

Comment:
Need more tests:
 - for rescaling.
 - to test lesser alphas / reservoir sizes.